app/views/calculators/adjustable_income/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<%
+  form = local_assigns[:form]
+  errors = form&.errors
+%>
+
+<h2 id="calculator">Estimate how much you could get</h2>
+
+<%= render 'calculators/error_summary', messages: errors&.messages if form&.invalid? %>
+
+<form action="/adjustable-income/estimate#form" method="get">
+  <%= render 'calculators/form_input_currency',
+             field: :pot,
+             value: form&.pot,
+             label: 'Amount you’ll have in your pot',
+             error_messages: errors&.get(:pot) %>
+
+  <%= render 'calculators/form_input_hidden',
+             field: :desired_monthly_income,
+             value: form&.desired_monthly_income %>
+
+  <%= render 'calculators/form_input_number',
+             field: :age,
+             value: form&.age,
+             label: 'Your age when you plan to retire',
+             error_messages: errors&.get(:age) %>
+
+  <%= render 'calculators/form_input_submit' %>
+</form>
+
+<% if form %>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    dataLayer.push({
+      'event': 'CalculatorEstimate',
+      'valid': <%= form.valid? %>
+      <% if form.valid? %>
+      , 'pot': <%= form.pot.to_i %>
+      , 'age': <%= form.age.to_i %>
```

Comment:
Having said that, `monthly_drawdown_amount` isn't available in this context. We can send a separate event to the datalayer on the `_estimate.html.erb` partial though:

```erb
<script>
  window.dataLayer = window.dataLayer || [];
  dataLayer.push({
    'event': 'CalculatorEstimate',
    'pot_payout': monthly_drawdown_amount.to_i
  });
</script>
```